cmd/batchDataExporter/main.go

```diff
@@ -0,0 +1,177 @@
+package main
+
+// goswagger code for spec discovery.yml is generated in ../localDiscovery
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	runtimeClient "github.com/go-openapi/runtime/client"
+	"github.com/go-openapi/strfmt"
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/postgres"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/push"
+	"github.com/rs/zerolog"
+
+	"github.com/iryonetwork/wwm/gen/storage/client"
```

Comment:
could not import github.com/iryonetwork/wwm/gen/storage/client (cannot find package "github.com/iryonetwork/wwm/gen/storage/client" in any of:
	/tmp/tmp.9iNHKsROgt/src/github.com/iryonetwork/wwm/vendor/github.com/iryonetwork/wwm/gen/storage/client (vendor tree)
	/usr/local/go/src/github.com/iryonetwork/wwm/gen/storage/client (from $GOROOT)
	/tmp/tmp.9iNHKsROgt/src/github.com/iryonetwork/wwm/gen/storage/client (from $GOPATH))